src/board.c

```
@@ -640,6 +550,9 @@ void Board_HandleLtc6804Status(LTC6804_STATUS_T status) {
             break;
         default:
             Board_Println("Entered default case in Board_HandleLtc6804Status(). You should never reach here");
+#ifdef FSAE_DRIVERS
+            Error_Assert(ERROR_CONTROL_FLOW, msTicks);
```

Why is the the Error_Assert(ERROR_CONTROL_FLOW, msTicks) statement inside an FSAE_DRIVERS flag? I talked about this control flow error with Eric and Skanda and they thought it was a good idea. Adding the control flow errors will help us catch bugs by failing fast.

---

This is because the ERROR_CONTROL_FLOW enum value is also ifdef-guarded. Can change this if the hooligans want the value as well.

---

I think the hooligans also want ERROR_CONTROL_FLOW,